### ISSUE TITLE
Use `change` event for Checkbox instead of `click`

### ISSUE THREAD

**Issue by zepumph** [phetsims/sun#574]
From https://github.com/phetsims/gravity-force-lab/issues/257 and https://github.com/phetsims/a11y-research/issues/156, it seems like we should investigate using the `change` event instead of the `click` event for a11y interaction with our checkboxes. I won't be able to work on this too soon, but feel like I should give a bit of back story.

We were finding that when NVDA selects the label for a checkbox, firing it triggers a click event from the label and a click event from the input. This results in two events that bubble up to the scenery event listener/dispatcher. We thought this behavior buggy, but actually this is the same event structure that you get when you click the label of a checkbox with a mouse on Chrome. From this I wonder if we shouldn't just be more tolerant of this behavior, and use the change event instead of the click event.

**Comment by zepumph** [comment]
@jessegreenberg what do you think?

**Comment by jessegreenberg** [comment]
Interesting, does that indicate that https://github.com/phetsims/a11y-research/issues/156 is actually not a bug? What would be solved by using the change  event instead of the click event, does the change event behave differently?

**Comment by zepumph** [comment]
Yes, I added some change logging to our snippet, note here that mouse clicking the label (or NVDA labeled clicking) results in click events from the label and the input, but ONLY one change event from the input.

```html
<!DOCTYPE html>
<html>
<body>
<div id='parent'>
  <p>something before</p>
  <label data-trail-id="input" id="label" for="input">Constant Size</label>
  <input data-trail-id="input" id="input" type="checkbox">
  <p>something after</p>
</div>
</body>
<script type="text/javascript">
  const input = document.getElementById( 'input' );
  const label = document.getElementById( 'label' );
  const parent = document.getElementById( 'parent' );

  input.addEventListener( 'click', () => {
    console.log( 'click - input', input.checked );
  } );
  label.addEventListener( 'click', () => {
    console.log( 'click - label', input.checked );
  } );
  parent.addEventListener( 'click', () => {
    console.log( 'click - parent', input.checked );
  } );
  input.addEventListener( 'change', () => {
    console.log( 'change - input', input.checked );
  } );
  label.addEventListener( 'change', () => {
    console.log( 'change - label', input.checked );
  } );
  parent.addEventListener( 'change', () => {
    console.log( 'change - parent', input.checked );
  } );
</script>
</html>

**Comment by jessegreenberg** [comment]
Interesting, thanks! In that case using the `change` event seems very reasonable to me.

**Comment by jessegreenberg** [comment]
I was hoping this would be a quick change but since Checkbox uses ButtonListener (which is what adds the `click` listener) it will take a little more thought. We likely don't want to change ButtonListener, as continuing to use `click` for buttons makes sense.

**Comment by marlitas** [comment]
Meeting 11/5
- We believe this issue is stale. @jessegreenberg will investigate

**Comment by jessegreenberg** [comment]
Reviewing the history, `excludeLabelSiblingFromInput` when we thought this was an NVDA bug. After, we learned that it is the expected behavior and so we wanted to remove `setExcludeLabelSiblingFromInput` and have checkbox use a `change` event instead. So this is not stale.

**Comment by zepumph** [comment]
Also noting that PhET-iO is in the area: https://github.com/phetsims/sun/issues/914
